account_invoice_secondary_unit/models/account_invoice_line.py

```diff
@@ -86,3 +87,9 @@ def _onchange_product_id(self):
             self.secondary_uom_qty = 1.0
             self.onchange_secondary_uom()
         return res
+
+    @api.constrains("secondary_uom_id","secondary_uom_qty")
+    def _check_secondary_uom_id(self):
+        for line in self:
+            if line.secondary_uom_qty and line.secondary_uom_id == False:
```

Comment:
if line.secondary_uom_qty > 0 and line.secondary_uom_id == False:

Reply:
Sorry , this line is not in the current commit.
Please watch file changes in the latest commit.